BSDmakefile

```diff
@@ -1,5 +1,5 @@
 #
-#  Copyright (C) CFEngine AS
+#  Copyright 2017 Northern.tech AS
```

Comment:
Why do you delete the old copyright? Who had copyright before 2017? Does the official company name contain a dot?

I would expect something like:

```
Copyright 2017-present Northern Tech AS
Copyright 2008-2017 CFEngine AS
```